mailo/src/test/scala/mailo/PersistenceSpec.scala

```diff
@@ -72,105 +64,100 @@ class MockedData extends MailData {
     Future(Right(MailRawContent("ciao", partials = Map("name" -> "claudio"))))
 }
 
-class PersistenceSpec
-  extends TestKit(ActorSystem("testSystem"))
-  with ImplicitSender
-  with WordSpecLike
-  with Matchers
-  with BeforeAndAfterAll
-  with BeforeAndAfterEach
-  with ScalaFutures {
-  import Eventually._
+class PersistenceSpec extends {
+  val system: ActorSystem = ActorSystem("testSystem")
+} with munit.FunSuite with TestKitBase with ImplicitSender {
 
   implicit def executionContext: ExecutionContext = system.dispatcher
- 
+
   override def afterAll: Unit = {
     TestKit.shutdownActorSystem(system)
   }
 
-  override def beforeEach(): Unit = {
-    val tp = TestProbe()
-    tp.send(StorageExtension(system).journalStorage, InMemoryJournalStorage.ClearJournal)
-    tp.send(StorageExtension(system).snapshotStorage, InMemorySnapshotStorage.ClearSnapshots)
-  }
-
-  implicit val defaultPatience =
-      PatienceConfig(timeout = Span(20, Seconds), interval = Span(5, Seconds))
-
-  "persistence actor" should {
-    "properly deliver email messages" in {
-      val state = new ConcurrentLinkedQueue[SimpleMail]()
-      val emailSender = new EmailSender(new MockedData, new MockedClient(state))
-      val loggingActor = system.actorOf(LoggingActor.props())
-      val emailPersistanceActor = system.actorOf(EmailPersistanceActor.props(emailSender, loggingActor))
-
-      val mail1 = Mail(
-        to = "mailo@buildo.io",
-        from = "Mailo mailo@buildo.io",
-        subject = "1",
-        templateName = "mail.html",
-        params = Map.empty,
-        tags = List("test")
-      )
-
-      val mail2 = mail1.copy(subject = "2")
-
-      emailPersistanceActor ! SendEmail(mail1)
-      expectMsg(Queued)
-
-      emailPersistanceActor ! SendEmail(mail2)
-      expectMsg(Queued)
-
-      eventually(timeout(Span(5, Seconds))) {
-        state.size should be(2)
-        info(s"${state.size} messages sent")
+  test("persistence actor should properly deliver email messages") {
+    val state = new ConcurrentLinkedQueue[SimpleMail]()
+    val emailSender = new EmailSender(new MockedData, new MockedClient(state))
+    val loggingActor = system.actorOf(LoggingActor.props())
+    val emailPersistanceActor =
+      system.actorOf(EmailPersistanceActor.props(emailSender, loggingActor))
+
+    val mail1 = Mail(
+      to = "mailo@buildo.io",
+      from = "Mailo mailo@buildo.io",
+      subject = "1",
+      templateName = "mail.html",
+      params = Map.empty,
+      tags = List("test"),
+    )
+
+    val mail2 = mail1.copy(subject = "2")
+
+    emailPersistanceActor ! SendEmail(mail1)
+    expectMsg(Queued)
```

Comment:
This is failing in the CI, didn't investigate so much but we might try to specify duration like `expectMsg(durationo)()`. I see the `defaultPatience` is removed.

![image](https://user-images.githubusercontent.com/2476080/77770211-3277fc80-7045-11ea-87d7-c6127061e54a.png)

Reply:
I think that's due to the change to the inmemory module of akka persistence (I've dropped the external library since it's not compatible with akka 2.6 and I'm using the internal one).
I'm aware of this and I'm trying to fix it (I can reproduce locally)

Reply:
(patience was a ScalaTest thing, used for the `eventually` part which I have removed)

Reply:
ok, fixed (it was a misconfiguration)